src/embed_tests/InitializeTest.cs

```diff
@@ -10,7 +10,29 @@ namespace Python.EmbeddingTest
     public class InitializeTest
     {
         [Test]
-        public static void Test()
+        public static void LoadSpecificArgs()
+        {
+            var args = new[] { "test1", "test2" };
+            using (new PythonEngine(args))
+            using (var argv = new PyList(Runtime.Runtime.PySys_GetObject("argv")))
+            {
+                Assert.That(argv[0].ToString() == args[0]);
+                Assert.That(argv[1].ToString() == args[1]);
+            }
+        }
+
+        [Test]
+        public static void LoadDefaultArgs()
+        {
+            using (new PythonEngine())
+            using (var argv = new PyList(Runtime.Runtime.PySys_GetObject("argv")))
+            {
+                Assert.That(argv.Length() != 0);
```

Comment:
Why? If it is `null`, this will fail with a `NullReferenceException`, perfectly fine.